lib/kittikrb/cursor.rb

```diff
@@ -103,6 +103,37 @@ def move_to(x, y)
         self
       end
 
+      def move_by(x, y)
+        @x += x.floor
+        @y += y.floor
```

Comment:
This method doesn't work. You should detect moving direction and call relative method (left, up, etc...).